api/external_resources/views.py

```diff
@@ -0,0 +1,21 @@
+from rest_framework import viewsets
+from rest_framework.permissions import IsAuthenticated
+
+from .models import ExternalResources
+from .serializers import ExternalResourcesSerializer
+
+
+class ExternalResourcesViewSet(viewsets.ModelViewSet):
+    serializer_class = ExternalResourcesSerializer
+    permission_classes = [IsAuthenticated]
```

Comment:
Should this need feature update permission? Or are we making it open by choice?

Reply:
Good question, maybe this is an option, or maybe create a new permission for this?